Svc/FileManager/FileManager.cpp

```diff
@@ -62,8 +62,9 @@ namespace Svc {
   {
     Fw::LogStringArg logStringDirName(dirName.toChar());
     this->log_ACTIVITY_HI_CreateDirectoryStarted(logStringDirName);
+    bool errorIfDirExists = true;
     const Os::FileSystem::Status status =
-      Os::FileSystem::createDirectory(dirName.toChar());
+      Os::FileSystem::createDirectory(dirName.toChar(), errorIfDirExists);
```

Comment:
Was the original implementation to error on exists?

Reply:
Yes! Not having it made a couple UTs fail and that's what prompted me adding it to the interface as an optional parameter